src/tsl/ordered_set.h

```diff
@@ -24,6 +24,12 @@
 #ifndef TSL_ORDERED_SET_H
 #define TSL_ORDERED_SET_H
 
+#ifdef _MSC_VER
+#if (_MSC_VER <= 1800)
+#include <xkeycheck.h>
```

Comment:
Perhaps the following will work:

```c++
#if defined(_MSC_VER) && (_MSC_VER <= 1800) && !defined(_ALLOW_KEYWORD_MACROS )
#define _ALLOW_KEYWORD_MACROS 1 
#endif
```

Reply:
It seems I need to find another ordered_map implementation anyway :(